framework/src/modules/chain/block_processor_v0.js

```diff
@@ -126,7 +127,8 @@ class BlockProcessorV0 extends BaseBlockProcessor {
 					blockBytes,
 				}), // validate common block header
 			data => this.blocksModule.verifyInMemory(data),
-			({ block }) => this.dposModule.verifyBlockForger(block),
+			({ block }) =>
+				this.dposModule.verifyBlockForger(block, delegateListRoundOffset),
```

Comment:
delegateListRoundOffset is kind of an option to the method. So maybe if pass it inside an object `this.dposModule.verifyBlockForger(block, {delegateListRoundOffset}),` would be more future proof? But we can also do that when there is a new option that comes up so we can leave it as it is now. I don't mind either way.

Reply:
I thought about that but for apply/undo as we have two params as options. Gonna update that as well